src/Skoruba.IdentityServer4.Admin/Helpers/Messaging/EmailSender.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
```

Comment:
This would be good to have in a shared project between both the Admin and the STS.Identity project.  This was pulled in from the STS.Identity project.